docs/reference/index-modules.asciidoc

```diff
@@ -234,6 +234,12 @@ specific index module:
     The length of time that a <<delete-versioning,deleted document's version number>> remains available for <<index-versioning,further versioned operations>>.
     Defaults to `60s`.
 
+  `index.default_pipeline`::
+
+    The default <<ingest,ingest node>> pipeline for this index. Index requests will fail
+    if the default pipeline is set and the pipeline does not exist. The default may be
+    ignored per request by setting the request parameter `pipeline=_none`.
```

Comment:
I suggestion something like this in place of the last sentence:

```
The default may be overridden using the `pipeline` parameter. The special pipeline name `_none` indicates no ingest pipeline should be run.
```

The point is to show that pipeline can be specified in general to override the default, and that `_none` is just a special pipeline name.